users/lamad/index.md

```diff
@@ -6,12 +6,33 @@ author: Adam Lam
 Adam Lam
 ================================
 
-lamad@oregonstate.edu
+<img src="{{ site.url }}users/harterj/images/jrh-sit.jpg" height="375" width="600">
```

Comment:
Looks like you may be adding a picture of Jackson?

Reply:
@aaronjamesreynolds do you not want a picture of Jackson on your page? 😁